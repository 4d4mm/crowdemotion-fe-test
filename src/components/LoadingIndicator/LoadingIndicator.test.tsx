import React from "react";
import { mount } from "enzyme";
import LoadingIndicator from "./LoadingIndicator";

describe("LoadingIndicator", () => {
  it("renders loading indicator with default ui", () => {
    const renderedLoadingIndicator = mount(<LoadingIndicator />);
    expect(renderedLoadingIndicator).toMatchSnapshot();
  });
});
