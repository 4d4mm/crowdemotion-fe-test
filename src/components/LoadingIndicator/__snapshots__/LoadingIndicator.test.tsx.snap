// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator renders loading indicator with default ui 1`] = `
<LoadingIndicator>
  <WithStyles(ForwardRef(CircularProgress))>
    <ForwardRef(CircularProgress)
      classes={
        Object {
          "circle": "MuiCircularProgress-circle",
          "circleDisableShrink": "MuiCircularProgress-circleDisableShrink",
          "circleIndeterminate": "MuiCircularProgress-circleIndeterminate",
          "circleStatic": "MuiCircularProgress-circleStatic",
          "colorPrimary": "MuiCircularProgress-colorPrimary",
          "colorSecondary": "MuiCircularProgress-colorSecondary",
          "indeterminate": "MuiCircularProgress-indeterminate",
          "root": "MuiCircularProgress-root",
          "static": "MuiCircularProgress-static",
          "svg": "MuiCircularProgress-svg",
        }
      }
    >
      <div
        className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
        role="progressbar"
        style={
          Object {
            "height": 40,
            "width": 40,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx={44}
            cy={44}
            fill="none"
            r={20.2}
            strokeWidth={3.6}
            style={Object {}}
          />
        </svg>
      </div>
    </ForwardRef(CircularProgress)>
  </WithStyles(ForwardRef(CircularProgress))>
</LoadingIndicator>
`;
